react/src/index.js

```diff
@@ -86,22 +91,29 @@ class App extends Component {
       }
     };
 
+    // new PerformanceObserver(entryList => {
```

Comment:
this was for debugging Largest Contentful Paint not getting set, but leaving it in somehow caused errors once it was in prod + tda was running

leave as commented as we're still fixing LCP, may need it.